src/pages/dummy-voting.tsx

```diff
@@ -0,0 +1,29 @@
+import { header } from "@fonts";
+import { useState } from "react";
+import Bg from "~/components/background";
+import { api } from "~/utils/api";
+
+const createVote = api.voteTools.create.useMutation({})
+
+export default function dumVote() {
+  const [nim, setNIM] = useState('')
+
+  return (
+    <>
```

Comment:
ini penempatannya salah yaa. terus sebenernya be ga perlu bikin react componentnya, kecuali buat nyoba mah silakan.